deployer/src/deployment/run.rs

```diff
@@ -375,12 +377,12 @@ async fn load(
             if response.success {
                 Ok(())
             } else {
-                error!(error = %response.message, "failed to load service");
+                error!(error = %response.message, "Failed to load service");
```

Comment:
Nit: IMO we should be consistent with the casing of error messages (and potentially other traces too, but at least error messages), and lowercase is the norm (e.g. rust std uses lowercase without trailing punctuation).